doc/api/tls.markdown

```diff
@@ -162,7 +162,10 @@ automatically set as a listener for the [secureConnection][] event.  The
     the `key`, `cert` and `ca` options.)
 
   - `key`: A string or `Buffer` containing the private key of the server in
-    PEM format. (Could be an array of keys). (Required)
+    PEM format. It can also be an array of keys. The array can either be of
+    just keys or if you have different passphrases for the keys, then the
+    array elements can be of the form `{pem: key, passphrase: passphrase}` and
+    the keys should use different algorithms. (Required)
```

Comment:
Ah, I see. I think this is worded a bit confusingly. How about rewording it like this?

```
A string or `Buffer` containing the private key of the server in
PEM format. To support multiple keys using different algorithms, an array
can be provided. It can either be a plain array of keys, or an array of
objects in the form of {pem: key, passphrase: passphrase}. (Required)
```